### ISSUE TITLE
[Fleet]: User is re-directed to `Apply a Policy` section on clicking on  `Add, remove, upgrade, or reassign agents to a policy ` link on `Elastic Agent Policies` Guide

### ISSUE THREAD

**Issue by prachigupta-qasource** [elastic/observability-docs#2310]
**Description:**

- User is re-directed to `Apply a Policy` section on clicking on  `Add, remove, upgrade, or reassign agents to a policy ` link on `Elastic Agent Policies` Guide

**Link:**

- https://www.elastic.co/guide/en/fleet/current/agent-policy.html

**Screen Recording:**

https://user-images.githubusercontent.com/97870262/199660105-61101f77-6302-4af3-ba2f-7d3af810048f.mp4

**Comment by amolnater-qasource** [comment]
Reviewed & mention to @dedemorton

**Comment by dedemorton** [comment]
Good catch. Probably an artifact of a backport/merge gone wrong. :-) 

Fix is up here: https://github.com/elastic/observability-docs/pull/2316

**Comment by prachigupta-qasource** [comment]
Hi @dedemorton,

We have revalidated the reported doc at link:

- https://www.elastic.co/guide/en/fleet/current/agent-policy.html

We have observed that the issue is fixed now.

-  `Add, remove, upgrade, or reassign agents to a policy` link is removed from the `Elastic Agent Policies` Guide.

**Screenshot:**

![image](https://user-images.githubusercontent.com/97870262/199886350-3ef2f59b-7697-45fe-93c8-4c91e95828b2.png)

Hence, marking this as QA: Validated.

Thanks!
